test/qunit/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <script src="../../dist/js-reporters.js"></script>
+    <link href="../../node_modules/qunitjs/qunit/qunit.css" rel="stylesheet" type="text/css"/>
+    <script src="../../node_modules/qunitjs/qunit/qunit.js"></script>
+    <script src="tests.js"></script>
+
+    <script>
+        var adapter = new JsReporters.QUnitAdapter();
+
+
+        var tapReporter = new JsReporters.TapReporter(adapter);
```

Comment:
I found it useful for testing, especially because you can toggle the debug level in the chrome dev tools. I think we want automated tests at some point.